kolibri/core/auth/test/test_morango_integration.py

```diff
@@ -1053,3 +795,94 @@ def assert_existence(self, server, kind, assignment_id, should_exist=True):
             ), "Assignment {assignment_id} should exist on server {server}!".format(
                 assignment_id=assignment_id, server=server
             )
+
+
+@unittest.skipIf(
+    not os.environ.get("INTEGRATION_TEST"),
+    "This test will only be run during integration testing.",
+)
+class SingleUserSyncRegressionsTestCase(TestCase):
+    @multiple_kolibri_servers(2)
+    def test_facility_user_conflict_syncing_from_tablet(self, servers):
+        self._test_facility_user_conflict(servers, True)
+
+    @multiple_kolibri_servers(2)
+    def test_facility_user_conflict_syncing_from_laptop(self, servers):
+        self._test_facility_user_conflict(servers, False)
+
+    def _test_facility_user_conflict(self, servers, tablet_is_client):
```

Comment:
This one has just been moved down to this new class (rather than being stuck in the clutter of the assignment testing class).